extensions/tika/runtime/src/main/java/io/quarkus/tika/graalvm/CleanerNotSupportedSubstitution.java

```diff
@@ -1,4 +1,4 @@
-package io.quarkus.tika.runtime.graal;
+package io.quarkus.tika.graalvm;
```

Comment:
This shouldn't be reverted.

Reply:
Resolved in https://github.com/quarkusio/quarkus/pull/14457/commits/b09b7085a2622c3489b7b902f1dcd5805b5799e8